src/tyre.ml

```diff
@@ -113,6 +134,9 @@ module Infix = struct
 end
 include Infix
 
+let (let+) x f = map f x
+let (and+) x y = seq x y
+
```

Comment:
Those should probably be in the `Infix` module as well.

Reply:
Some people like to `open M.Infix` to benefits from the operator without bringing lot's of generically named functions.